### DISCUSSION TITLE
Shell.TitleView on Windows

### DISCUSSION THREAD

**Discussion by Cywizz** [dotnet/maui#8222] [Q&A]
In Windows, the TitleView has a "See More" button which expands the view.
Is there anyway that you can expand/collapse this via code?

**Comment by Laradius** [comment]
Have you maybe figured this out?

I have added it as a feature request

https://github.com/dotnet/maui/issues/8597

**Reply by Cywizz** [reply]
Not yet...I parked the issue for now. Thanks for submitting the feature request
